pkg/appmanager/outputConfig.go

```diff
@@ -27,10 +27,21 @@ import (
 // Dump out the Virtual Server configs to a file
 func (appMgr *Manager) outputConfig() {
 	appMgr.resources.Lock()
-	appMgr.outputConfigLocked()
+	if appMgr.as3Mode {
```

Comment:
```suggestion
	if appMgr.agent == "as3" {
```
In accordance with the parameter modification.

Reply:
Done